### ISSUE TITLE
Enhanced support for the EU store

### ISSUE THREAD

**Issue by sybrew** [sybrew/the-seo-framework#437]
The [European store just opened](https://eu.theseoframework.com/get-license/)! Now, you can finally use other payment methods than just a credit card.

It's already supported by the Extension Manager thanks to some rerouting of the API on our servers.
This isn't super efficient, as both your website as our server has to handle multiple requests.

Moreover, the "My Account" link doesn't lead to the right dashboard when connected via an EU-API key.

So, to summarize, this needs to be done:
- Reroute API endpoints from the start by detecting the API key header.
- Change interface links accordingly, depending on the API key used.
- Add an indicator in the dashboard, telling whether it's using a "Global" or "Euro" connection.
